src/distributions/mod.rs

```diff
@@ -271,6 +271,18 @@ impl<'a, T, D: Distribution<T>> Distribution<T> for &'a D {
 /// than the guarantee some value (`0.0`) is never generated. That makes an open
 /// interval a nicer choice.
 ///
+/// Consider using `Rng::gen_range(0.0, 1.0)` if you really need a half-open
+/// interval (as the ranges use a half-open interval), and has the same
+/// performance:
```

Comment:
I think the second part of the sentence is missing a word. Maybe just make it "It has the same performance."?